app/src/middleware/apiTracker.js

```diff
@@ -87,20 +113,24 @@ const initializeApiTracker = app => {
     return req._ts ? req._ts : '0';
   });
 
+  morgan.token('count', req => {
+    return req.body && Array.isArray(req.body) ? req.body.length : '0';
```

Comment:
Is that 0 meant to be a string or an integer?

Reply:
It has to be a string, if it is a number, then morgan treats it as a boolean and places '-' as the value.